libraries/botbuilder-dialogs-adaptive/src/generators/languageGeneratorManager.ts

```diff
@@ -34,7 +34,7 @@ export class LanguageGeneratorManager {
         const resources = await this._resourceExporer.getResources('lg');
         for (const resource of resources) {
             const generator = await this.getTemplateEngineLanguageGenerator(resource);
-            this.languageGenerator.set(resource.id().toLocaleLowerCase(), generator);
+            this.languageGenerator.set(resource.id(), generator);
```

Comment:
Should we still call `.toLowerCase()`?

Reply:
no... thats the bug


---
In reply to: [418757742](https://github.com/microsoft/botbuilder-js/pull/2160#discussion_r418757742) [](ancestors = 418757742)